$NetBSD$

#Use the correct compiler variable (CXX, not CC) and respect the environment
--- Makefile.orig	2017-02-11 22:31:35 UTC
+++ Makefile
@@ -1,5 +1,5 @@
-CC=gcc
-CFLAGS=-g -Wall -O2 -Wno-unused-function
+CC ?=		gcc
+CFLAGS ?=	-g -Wall -O2 -Wno-unused-function
 
 all:trimadap-mt
 
